tests/serverpod_test_server/test_e2e/cloud_storage_test.dart

```diff
@@ -1,22 +1,41 @@
 import 'dart:typed_data';
+import 'dart:async';
 
 import 'package:http/http.dart' as http;
 import 'package:serverpod_test_client/serverpod_test_client.dart';
 import 'package:serverpod_test_server/test_util/config.dart';
 import 'package:test/test.dart';
 
+Stream<int> streamBytes() async* {
+  int i = 0;
+  while (true) yield i++ % 256;
+}
+
+extension<T> on Stream<T> {
+  Stream<List<T>> inChunks(int chunkSize) async* {
```

Comment:
Nit: `inChunks(5)` reads like it would return 5 chunks, not chunks of size 5 to me.